devauth/devauth.go

```diff
@@ -124,6 +124,7 @@ type App interface {
 	ProvisionDevice(ctx context.Context, dev *model.Device) (err error)
 
 	GetTenantDeviceStatus(ctx context.Context, tenantId, deviceId string) (*model.Status, error)
+	SetExternalIdentity(context.Context, string, *model.ExternalDevice) error
```

Comment:
I did it mainly because of line-width. However, I see I've caused an inconsistency and should probably have named the arguments as for the rest of the interface definition.